### PR TITLE
Command to list sstables path in a node

### PR DESCRIPTION
**Pull request by sylvestor88**
Created a new PR with the changes suggested in the older PR (Getsstables #329)

### CODE REVIEW

File: ccmlib/cmds/node_cmds.py [hunk 1]
```diff
@@ -459,6 +460,40 @@ def run(self):
...
+class NodeGetsstablesCmd(Cmd):
+    def description(self):
+        return "Run getsstables to get absolute path of sstables in this node"
+
+    def get_parser(self):
+        usage = "usage: ccm node_name getsstables [options] [file]"
+        parser = self._get_default_parser(usage, self.description())
+        parser.add_option('-k', '--keyspace', type="string", dest="keyspace", default=None,
+                          help="The keyspace to use [use all keyspaces by default]")
+        parser.add_option('-c', '--column-families', type="string", dest='cfs', default=None,
```
Comment (ptnapoleon) [inline ccmlib/cmds/node_cmds.py]: Please use tables, instead of column families.

File: ccmlib/node.py [hunk 2]
```diff
@@ -812,6 +812,13 @@ def run_sstablemetadata(self, output_file=None, datafiles=None, keyspace=None, c
         if output_file == None:
             return results
 
+    def get_sstables(self, output_file=None, datafiles=None, keyspace=None, column_families=None):
+        cdir = self.get_install_dir()
+        sstablemetadata = common.join_bin(cdir, os.path.join('tools', 'bin'), 'sstablemetadata')
+        env = common.make_cassandra_env(cdir, self.get_path())
+        sstablefiles = self.__gather_sstables(datafiles=datafiles, keyspace=keyspace, columnfamilies=column_families)
+        print_(sstablefiles)
```
Comment (ptnapoleon) [inline ccmlib/node.py]: This command should return the sstablefiles, then the run() method in node_cmds.py should print out the returned value. This makes it more usable from other clients.

File: ccmlib/node.py [hunk 3]
```diff
@@ -812,6 +812,13 @@ def run_sstablemetadata(self, output_file=None, datafiles=None, keyspace=None, c
         if output_file == None:
             return results
 
+    def get_sstables(self, output_file=None, datafiles=None, keyspace=None, tables=None):
+        cdir = self.get_install_dir()
+        sstablemetadata = common.join_bin(cdir, os.path.join('tools', 'bin'), 'sstablemetadata')
```
Comment (ptnapoleon) [inline ccmlib/node.py]: I don't understand what's going on here. You never call `sstablemetadata`, or forward this variable

File: ccmlib/cmds/node_cmds.py [hunk 4]
```diff
@@ -459,6 +460,41 @@ def run(self):
...
+class NodeGetsstablesCmd(Cmd):
+    def description(self):
+        return "Run getsstables to get absolute path of sstables in this node"
+
+    def get_parser(self):
+        usage = "usage: ccm node_name getsstables [options] [file]"
+        parser = self._get_default_parser(usage, self.description())
+        parser.add_option('-k', '--keyspace', type="string", dest="keyspace", default=None,
+                          help="The keyspace to use [use all keyspaces by default]")
+        parser.add_option('-c', '--tables', type="string", dest='tables', default=None,
```
Comment (ptnapoleon) [inline ccmlib/cmds/node_cmds.py]: Can you make this short option `-t`? It doesn't make sense to have `-c` with `--tables`

File: ccmlib/cmds/node_cmds.py [hunk 5]
```diff
@@ -459,6 +460,41 @@ def run(self):
...
+                          help="Comma seperated list of tables to use (requires -k to be set)")
```
Comment (ptnapoleon) [inline ccmlib/cmds/node_cmds.py]: `separated` spelled wrong again

File: ccmlib/node.py (hunk 3 above)
Comment (sylvestor88) [inline ccmlib/node.py]: Those 3 statements shouldn't have been there. My bad. Have been removed.

File: ccmlib/cmds/node_cmds.py (hunk 4 above)
Comment (sylvestor88) [inline ccmlib/cmds/node_cmds.py]: Done

File: ccmlib/cmds/node_cmds.py (hunk 5 above)
Comment (sylvestor88) [inline ccmlib/cmds/node_cmds.py]: Checked

**Comment by ptnapoleon** [comment]
+1

**Pull request merged**